frontend/src/pam/mission/mission-item.tsx

```diff
@@ -0,0 +1,193 @@
+import { Accent, Button, Icon, logSoftError, Size, THEME } from '@mtes-mct/monitor-ui'
+import { Link } from 'react-router-dom'
+import { Divider, FlexboxGrid, Stack } from 'rsuite'
+import { formatMissionName } from './utils.ts'
+import MissionOpenByTag from './mission-open-by-tag.tsx'
+import { formatDateForFrenchHumans } from '../../utils/dates.ts'
+import MissionStatusTag from './mission-status-tag.tsx'
+import MissionCompletenessForStatsTag from './mission-completeness-for-stats-tag.tsx'
+import useIsMissionCompleteForStats from './use-is-mission-complete-for-stats.tsx'
+import styled from 'styled-components'
+import * as Sentry from '@sentry/react'
+import useLazyMissionExport from './export/use-lazy-mission-export.tsx'
+import useLazyMissionAEMExport from './export/use-lazy-mission-aem-export.tsx'
+
+interface MissionItemProps {
+  mission: Mission
+  prefetchMission: (missionId: string) => void
+}
+
+const ListItemWithHover = styled.div`
+  height: inherit;
+  background-color: ${THEME.color.cultured};
+  transition: background-color 0.3s ease;
+
+  &:hover {
+    background-color: ${THEME.color.blueGray25};
+  }
+`
+
+const MissionItem: React.FC<MissionItemProps> = ({mission, prefetchMission}) => {
+
+  const [getMissionReport] = useLazyMissionExport()
+  const [getMissionAEMReport] = useLazyMissionAEMExport()
+  const [exportLoading, setExportLoading] = useState<boolean>(false)
+
+  const [exportationCanBeDisplayed, setExportationCanBeDisplayed] = useState<boolean>(false)
+  const exportRapportEnabled = useIsMissionCompleteForStats(mission?.id)
+
+  const handleDownload = (missionExport?: MissionExport) => {
+    if (missionExport) {
+      try {
+        const content = missionExport?.fileContent
+        // Decode base64 string
+        const decodedContent = atob(content)
+
+        // Convert the decoded content to a Uint8Array
+        const uint8Array = new Uint8Array(decodedContent.length)
+        for (let i = 0; i < decodedContent.length; i++) {
+          uint8Array[i] = decodedContent.charCodeAt(i)
+        }
+
+        // Create a Blob from the Uint8Array
+        const blob = new Blob([uint8Array], { type: 'application/vnd.oasis.opendocument.text' })
+
+        // Create a temporary link element
+        const link = document.createElement('a')
+        link.href = window.URL.createObjectURL(blob)
+        link.download = missionExport.fileName
+
+        // Append the link to the document body and trigger a click event
+        document.body.appendChild(link)
+        link.click()
+
+        // Remove the link element from the document body
+        document.body.removeChild(link)
+      } catch (error) {
+        console.log('handleDownload error: ', error)
+        Sentry.captureException(error)
+      }
+    }
+  }
+
+  const exportMission = async (missionId, isAEMExport: boolean = false) => {
+    setExportLoading(true)
+
+    let { data, error, loading, called } = await getMissionReport({ variables: { missionId } })
```

Comment:
un petit test sur l'appel ou pas de getMissionReport :) 
 - data-testid sur le bouton
 - dans le test getByDataTestId and click
 - mock getMissionReport avec un vi.fn(), expect haveBeenCalled 
 ???!!!